src/explore-education-statistics-common/src/validation/serverValidations.ts

```diff
@@ -163,13 +163,22 @@ export function isServerValidationError(
 
 export function hasServerValidationError(
   error: Error,
-  errorMessage: string,
+  ...errorCodes: string[]
```

Comment:
Np - method replaced anyway, but I'd be interested to know why not to use spread params?  Always seemed a bit more flexible in use to me.

Reply:
I personally find they're generally less flexible as they have specific rules. They're particularly offending as they make it so that you can't just add new parameter to the end of the method args without going to effort to refactor all the existing callsites.

Of course, there are types of APIs that suit spread params, but don't think this kind of thing is particularly it.

Can understand using spread params more in languages where we have to do things like `new List()`, but in JS, there's very pretty much zero effort between doing the two (besides some square brackets):

```js
myFunction(['a', 'b', 'c']);
// vs
myFunction('a', 'b', 'c');
```

Reply:
Good stuff, thanks for explanation.